nvram.c

```diff
@@ -62,15 +62,19 @@ static void firmae_load_env()
 int nvram_init(void) {
     FILE *f;
 
-    PRINT_MSG("%s\n", "Initializing NVRAM...");
-    hc("NVRAM INIT");
+
 
     if (init) {
         PRINT_MSG("%s\n", "Early termination!");
         return E_SUCCESS;
     }
     init = 1;
-
+    PRINT_MSG("%s\n", "Initializing NVRAM...");
+    char * msg = "NVRAM INIT";
```

Comment:
It would probably be a lot cleaner and easier for you if these various strings were instead unique ints that you used as your magic values. Then you don't have to worry about them getting paged out